website/content/en/docs/building-operators/ansible/quickstart.md

```diff
@@ -32,8 +32,7 @@ This guide walks through an example of building a simple memcached-operator powe
 Make sure to define `IMG` when you call `make`:
```

Comment:
Would not this one has the goal to close the issue: https://github.com/operator-framework/operator-sdk/issues/4415?